lib/npm.js

```diff
@@ -162,7 +170,7 @@ export let createDependencyVersion = async ([name, version]) => {
  */
 export let createDependency = async ([name, version]) => {
 	let npmName = await createDependencyName(name)
-	let npmVersion = await createDependencyVersion([name, version])
+	let npmVersion = createDependencyVersion([name, version])
```

Comment:
is this missing?
```suggestion
	let npmVersion = await createDependencyVersion([name, version])
```

Reply:
This was missing yeah, good catch. I removed it as part of a warning I got in my editor about `createDependencyVersion` not needing to be an async function as it doesn't use await. I changed `createDependencyVersion` to be a normal function and removed this await call but it made the tests fail. When I went to revert my work I forgot to add this `await` back 🙈